openassessment/xblock/submission_mixin.py

```diff
@@ -113,9 +116,35 @@ def submit(self, data, suffix=''):  # pylint: disable=unused-argument
 
         status_tag = 'ENOMULTI'  # It is an error to submit multiple times for the same item
         status_text = self._(u'Multiple submissions are not allowed.')
+
         if not workflow:
             try:
-                submission = self.create_submission(student_item_dict, student_sub_data)
+
+                if self.teams_enabled:
+                    # a submission for a team generates matching submissions for all members
+                    submissions = self.create_team_submission(student_item_dict, student_sub_data)
+
+                    result = []
+
+                    for submission in submissions:
+                        status = True
+                        status_tag = submission.get('student_item')
+                        status_text = submission.get('attempt_number')
```

Comment:
Consider making this into a helper function, since the same block is also duped below